common/test/data/unicode_graded/README.md

```diff
@@ -0,0 +1 @@
+This is a very simple course for testing student responses reports for courses with unicode information.
```

Comment:
Neither of those have problems. This test course has problems with unicode in the display names, so we can test the CSV handling that.